jade/distribution.yaml

```diff
@@ -3650,6 +3650,51 @@ repositories:
       url: https://github.com/ros-visualization/rqt_robot_plugins.git
       version: master
     status: developed
+  rsv_balance:
+    doc:
+      type: git
+      url: https://github.com/robosavvy/rsv_balance.git
+      version: master
+    release:
+      packages:
+      - rsv_balance_description
+      - rsv_balance_msgs
+      url: https://github.com/robosavvy/rsv_balance.git
```

Comment:
Yes, that would be sufficient for documentation.

But I assume your repositories build on-top of each other. Therefore you might also want to remove the `source` blocks since the continuous integration jobs will fail if the dependencies are not available as Debian packages.